pep-0XXX-environment-markers.rst

```diff
@@ -0,0 +1,174 @@
+PEP: 0XXX
+Title: Environment Markers
+Version: $Revision$
+Last-Modified: $Date$
+Author: James Polley <jp@jamezpolley.com>
```

Comment:
I wouldn't worry about the Replaces line in this case - PEP 426 will eventually replace 345 entirely.